packages/javascript-sdk/src/index.ts

```diff
@@ -124,6 +125,7 @@ export {
   HttpClient,
   KbaCreateCallback,
   LocalStorage,
+  LoggerFunctions,
```

Comment:
Since this is just a type, can we export it in the above block that's dedicated to exporting just types?